### ISSUE TITLE
MS Office: items in file ribbon are reported twice

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#8390]
### Steps to reproduce:
1. Open MS Word, Excel or Power Point
2. Press alt to open the ribbon bar
3. Press left arrow to go to the ribbon "file"
4. Press down arrow multiple times

### Expected behavior:
NVDA should report the items only once (i.e. options etc.)

### Actual behavior:
NVDA reports the items twice (i.e. Options Options)

### System configuration:
NVDA version:
2018.2 RC3

NVDA Installed or portable:
Installed

Windows version:
Windows 10 1803 Spring creators update

Name and version of other software in use when reproducing the issue:
MS Office 2016

### Other questions:

Does the issue still occur after restarting your PC?
Yes

Have you tried any other versions of NVDA?
Yes, 2017.1, 2017.3, 2017.4

**Comment by Qchristensen** [comment]
As at NVDA 2018.4beta2 this issue still exists in Office 365 (version 16.0.11001.20093).  One observation is that with "Report Object Descriptions" unchecked in NVDA's Object Presentation settings, each File menu item is only reported once.

This issue also only affects the file menu, not the main ribbon sections (Home, Insert, etc), and also only the main items in the File menu ("Home", "New", "Open", "Info" etc) not once you tab to move into the say open task pane.

**Comment by Adriani90** [comment]
Closing as duplicate of #6740.
